### PR TITLE
Allow push to init a wiki repo

### LINKED ISSUE #1278: Cannot push a wiki when there is not any wiki page.
When created a repo, we can `git push -u origin master` to send the content, but cannot send content to the wiki repo

### PR DESCRIPTION
**Pull request by lunny**
resolved #1278

### CODE REVIEW

File: cmd/serv.go [hunk 1]
```diff
@@ -296,6 +296,13 @@ func runServ(c *cli.Context) error {
 		gitcmd = exec.Command(verb, repoPath)
 	}
 
+	if isWiki {
+		err = repo.InitWiki()
+		if err != nil {
```
Comment (appleboy) [inline cmd/serv.go]:
```go
if err := repo.InitWiki(); err != nil {
....
}
```

more readable?

Comment (bkcsoft) [reply cmd/serv.go]: Also a good idea to contain them (not that it matters _here_, but it's good practice)

**Comment by bkcsoft** [comment]
Other than that comment, LGTM

**Review by appleboy — APPROVED** [review]

**Comment by strk** [comment]
LGTM

File: cmd/serv.go (hunk 1 above)
Comment (lunny) [reply cmd/serv.go]: done.

**Comment by lunny** [comment]
let L-G-T-M work

**Pull request merged**